### PR TITLE
Update readme.cce

### PR DESCRIPTION
**Pull request by lxavier**
Change instruction to new KEsch env

### CODE REVIEW

File: mpiopenacc/readme.cce [hunk 1]
```diff
@@ -33,10 +39,14 @@ ldd mpiacc_f.exe | grep "not"
 
 ### C
 cc -hacc ../src/mpiacc_c.c -o mpiacc_c.exe
+# CC-1254 crayc: WARNING 
+#  The environment variable "CPATH" is not supported.
```
Comment (lxavier) [inline mpiopenacc/readme.cce:43]: Does anyone know why we get this warning ?

Comment (gppezzi) [inline mpiopenacc/readme.cce:43]: GCC/4.9.3-binutils-2.25 defines the CPATH variable and the Cray compiler apparently doesn't like it. 

Normally this warning should be harmless, if you get missing std c++ headers we might need to use the craypkgconfig tool to make sure that the Cray wrapper is using the correct include path when using this GCC module.

If needed, we might also consider removing this CPATH environment from the GCC module.

Comment (lxavier) [inline mpiopenacc/readme.cce:43]: ok fine for me, no need to change the module

File: mpiopenacc/readme.cce [hunk 2]
```diff
@@ -60,7 +70,8 @@ srun  --gres=gpu:1 --exclusive -n1 mpiacc_c.exe 128
 	# c[N-1]=12700.000000
 
 
-### C++
+### C++ 
+# Currently not working
```
Comment (lxavier) [inline mpiopenacc/readme.cce:74]: this is not working, but we don't need c++ and OpenACC so far

**Pull request merged**